provider/src/main/kotlin/org/gradle/kotlin/dsl/support/KotlinCompiler.kt

```diff
@@ -188,6 +188,12 @@ fun compilerConfigurationFor(messageCollector: MessageCollector, sourceFiles: It
     CompilerConfiguration().apply {
         addKotlinSourceRoots(sourceFiles.map { it.canonicalPath })
         addJvmClasspathRoots(PathUtil.getJdkClassesRootsFromCurrentJre())
```

Comment:
Is this line still required if `JDK_HOME` is set?

Reply:
I have no idea. Personally, I'd rather not mess with it. I can run the tests locally and see if it works without it if you'd like.

Reply:
I just ran `./gradlew check --parallel` without that line and pretty much all of the tests fail, so yes, it is still needed.